packages/fuel-indexer/src/executor.rs

```diff
@@ -172,6 +174,8 @@ pub fn run_executor<T: 'static + Executor + Send + Sync>(
                 return Err(IndexerError::KillSwitch);
             }
 
+            let _start = Instant::now();
```

Comment:
```suggestion
            #[cfg(feature = "metrics")]
            let start = Instant::now();
```

I think we should uniformly feature-gate the code.